### DISCUSSION TITLE
Is there an option to disable PR generation?

### DISCUSSION THREAD

**Discussion by doljae** [renovatebot/renovate#14794] [Request Help] [answered]
I want to **disable PR generation** and **use only the automatically updated dependency dashboard** function.

With this, I want to monitor only the latest version of the dependencies I am using.

Are there any options for this? If there is, I am curious how to set it up on Github and Github Enterprise.

**Comment by rarkins** [comment, marked as answer]
You can configure dependencyDashboardApproval=true

**Reply by doljae** [reply]
Thank you for kine reply.

Is the dependency dashboard automatically created to check all versions released after the current dependency?

For example, assuming that `dependency A` `1.0` is used in the current project and `1.1`, `1.2`, and `2.0` have been released since then, I wonder if it is possible to create a PR by selecting `1.1`, `1.2`, `2.0` on the dashboard.

**Reply by doljae** [reply]
And if there is an option to check the previous versions of the dependecy version using dashboard, please let me know.

**Reply by rarkins** [reply]
I think it would be best if you create a test repo with this setting and experiment yourself

**Reply by doljae** [reply]
OK I'll try, thank you for kind support!

**Reply by viceice** [reply]
Search the docs for `separate`

https://docs.renovatebot.com/faq/#separate-patch-releases-from-minor-releases

Then you get a pr for nearly each version.

you probably are interested in
- https://github.com/renovatebot/renovate/issues/6576

**Reply by doljae** [reply]
@viceice, Thank you for kind reply with references.
Maybe I can do what I want using the references you gave.😄
